script.js

```diff
@@ -1,4 +1,86 @@
+outputValue = `Player ${currentPlayer}, your chosen value is ${playerScore}! `;
+var gameMode = ""; //Modes: diceroll, getplayerinput, generateresult
+var currentPlayer = 1;
+var playerDiceRolls = [];
+var playerScore = 0;
+var outputValue = "";
+var allPlayerScores = [];
+
+function rollDice() {
+  var diceNumber = Math.ceil(Math.random() * 6);
+  return diceNumber;
+}
+
+function rollDiceForPlayer() {
+  var counter = 0;
+  while (counter < 2) {
+    playerDiceRolls.push(rollDice());
+    counter++;
+  }
+  return (outputValue = `Player ${currentPlayer}, your dice rolls are ${playerDiceRolls[0]} and ${playerDiceRolls[1]}. <br><br> Please choose the order of the dice by inputting 1 or 2.`);
```

Comment:
While this might work, it is not really a normal convention. 
You should assign outputValue its value first before returning outputValue